frontend/src/__mocks__/mockRunKF.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable camelcase */
+import { PipelineRunKF, RunStorageStateKF } from '~/concepts/pipelines/kfTypes';
+
+type MockRunKFParams = {
+  name?: string;
+  id?: string;
+};
+
+export const mockRunKF = ({
+  name = 'test-pipeline',
```

Comment:
I think this and the other mock function you have are probably derived from the "pipelines" mock, and I think these default values, along with the ones in `mockJobKF` can be more job/run oriented by calling the defaults "Test Job" or "Test Run". We could also probably make the `MockRunKFParams` optional if we set default values like we are here (same with the equivalent params in mockJobKF).